src/Wallet.ts

```diff
@@ -16,12 +16,26 @@ import {ValidationError} from './common/errors'
 class Wallet {
   readonly publicKey: string
   readonly privateKey: string
+  readonly classicAddress: string
+  readonly seed?: string
```

Comment:
What is the purpose in adding a `seed` instance variable to the wallet?

Reply:
In the case of creating a new Wallet (using `Wallet.create()`), I think it's helpful for users to know what `seed` was used that created a `Wallet` instance so they can reuse this seed to derive the same wallet in the future.

Reply:
Can you not create a wallet from the private key?